src/core/server/integration_tests/saved_objects/migrations/group3/dot_kibana_split.test.ts

```diff
@@ -414,15 +416,29 @@ describe('split .kibana index into multiple system indices', () => {
       const esClient = await getEsClient();
 
       const breakdownBefore = await getAggregatedTypesCountAllIndices(esClient);
-      expect(breakdownBefore).toMatchSnapshot('before migration');
+      expect(breakdownBefore).toEqual({
```

Comment:
Expecting a definitive result is definitely better than matching my snapshot! Snapshots are 'easy' to update, whether what they represent is right or not. Asserting against what we know the response should be is stricter, and we'll pay more attention to what went wrong. 
Nice!